### ISSUE TITLE
Resnet50 Pytorch Model Accuracy Loss 

### ISSUE THREAD

**Issue by mai202** [pytorch/glow#4674]
Hello,
I took the resnet50 PyTorch model from torchvision and exported to ONNX. When I ran it using image-classifier on first 1000 images of imagenet data set, i am seeing almost 20% accuracy loss from the resnet50 caffe2 model (on same 1000 images). It makes me wonder if the options i am using for running pytorch model is not correct. I am using "-use-imagenet-normalization"  "-compute-softmax" (pytorch model does not softmax in the end) and "-image-mode=0to1". Does input to pytorch model gets normalized in the same way as caffe2 model?

What can I check to debug this further?

**Comment by leejaymin** [comment]
I have the same issue. In my case, resnet18 shows 20% accruacy drop comparing with pytorch inference with imagenet2012 validation set(50,000). 
Do Caffe2 resnet models show the correct accuracy?
I wonder too.

**Comment by mai202** [comment]
Yes caffe2 resnet model shows correct accuracy.

**Comment by leejaymin** [comment]
Thank you for sharing your results.
I also need to get the correct results with ONNX models.

**Comment by mai202** [comment]
Thanks for sharing your result. It does match with my results on resnet50.

One other thing, i noticed Pytorch model is equivalent to version 2 of resnet. 

If you directly take the onnx models for onnx github, you will see v2 models showing the same kind of loss and v1 works fine.
https://github.com/onnx/models/tree/master/vision/classification/resnet/model

So something to do running v2 models on glow then?
